examples/basics/plotting/grid_uneven_col.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
+columns ->
+  0     1    2
+0 +-----+----+------+
+  | 1   |  2 |  2   |
+1 +-----+----+------+
+  |   3 |  3 |   4  |
+2 +-----+----+------+
```

Comment:
if "columns" goes 0,1,2 then "rows" should only go 0,1

Also the numbers should probably be centered on the rows and columns

And the "1" in the upper left does not line up with the "3" in the lower left